content/src/content/jcr_root/apps/core/wcm/components/list/v1/list/clientlibs/editor/js/list.js

```diff
@@ -18,7 +18,7 @@
 
     $(document).on("foundation-contentloaded", function(e) {
         $(".cmp-list--editor coral-select.cq-dialog-dropdown-showhide", e.target).each(function(i, element) {
-            var target = $(element).data("cqDialogDropdownShowhideTarget");
+            var target = $("<div>").text($(element).data("cqDialogDropdownShowhideTarget")).html();
             if (target) {
```

Comment:
You may want to use window.Granite.UI.Foundation.Utils.sanitizeHtml() instead of this which looks much cleaner although it works a bit differently. Could you check if it's OK? IMHO it should do the job.

Reply:
Yes, checked and it works